routes.js

```diff
@@ -56,13 +56,29 @@ router.post("/postWorkerDetail", async (req, res) => {
   });
   try {
     const workerSave = await workerPost.save();
+    const dataKeys = Object.keys(req.body);
```

Comment:
Use mongoose to get the property instead of using Object.keys, refer mongoose docs

https://www.mongodb.com/docs/manual/reference/operator/aggregation/group/